code/common/datadog.q

```diff
@@ -0,0 +1,41 @@
+// DATADOG CHECKS
+
+//Create datadog namespace
+\d .dg
+
+//Find the config file created during set up containing the port set by user in setenv. If it doesn't exist default to 8125.
+dgconfigfile:@[value;`dgconfigfile;hsym first .proc.getconfigfile"dgconfig.txt"]
+//sets dogstatsd_port to the port defined by dgconfigfile
+$[`dgconfig.txt in key hsym `$getenv[`KDBAPPCONFIG];value each read0 dgconfigfile;dogstatsd_port:8125]
```

Comment:
What does the config file look like? This makes it seem that it is a text file containing q commands?
```
value each read0 dgconfigfile
```
This line still suffers from the same issue from before where users can rename the config file but this stills expects `dgconfig.txt` I think you want this:
```suggestion
$[not()~key dgconfigfile;value each read0 dgconfigfile;dogstatsd_port:8125]
```
I would say the variable `dgconfigfile` should be the full path (not relative).

Reply:
Sorry these functions have been moved to the datachecks.q file in datadog folder. I'm not sure how they ended up still being in this file, I thought I'd removed them